ChangelogGenerator/ChangelogGenerator/Program.cs

```diff
@@ -23,6 +23,9 @@ class Options
         [Option('m', "Milestone", Required = true, HelpText = "Milestone to get issues from")]
         public string Milestone { get; set; }
 
+        [Option('l', "RequiredLabel", Required = true, HelpText = "Hide all issues without this label")]
```

Comment:
a more accurate help text would be "Show only those issues from the selected milestone that have this label"